suppliers/data-destruction-contract-clauses-long-format-appendix.md

```diff
@@ -0,0 +1,31 @@
+---
+category: Data Destruction
+expires: 2019-12-31
+---
+# Long format appendix
+
+The current draft of the MOJ commodity long format data destruction appendix. The appendix is a dependancy of the long format clause itself.
+
+`Highlighted` words indicate potential requirement for contextual change, requirement of definition and so on.
+
+## Appendix
+
+While all applicable and proportional steps must be defined by the context of the dataset(s) and/or system(s), the following standards or guidelines establish the *minimum* requirements that **must** be followed and applied accordingly:
```

Comment:
Only observation here is that it isn't immediately clear whether you mean to follow _all_ of the following bullet points, or (as I suspect is the intention) that the following bullet points _may_ be relevant, depending on circumstance, and offer a sensible baseline.

Reply:
The next bit is a bunch of standards links, the bit towards the end L21 / L26 define the sensible baseline whether tech or paper. Does the whole thing together clear it up or worth updating this bit?

Reply:
I doubt it'll cause that much confusion in the grand scheme of things, but I can see how someone turns this into 'do all five of these requirements', rather than tailoring to meet specific situation, as I think it does read a bit like 'all these five points are required'.

Reply:
Phrasing updated, lifted from elsewhere which offers more clarity.